docs/sql-programming-guide.md

```diff
@@ -1302,8 +1302,9 @@ Configuration of in-memory caching can be done using the `setConf` method on `Sp
 
 ## Other Configuration Options
 
-The following options can also be used to tune the performance of query execution. It is possible
-that these options will be deprecated in future release as more optimizations are performed automatically.
+The following options can also be used to tune the performance of query execution and attaching
```

Comment:
I don't think this new option belongs in this section. It has nothing to do with performance and this description now sounds weird. A separate section for it would be better, even if it's the only option there.